src/components/Badge/Badge.test.tsx

```
@@ -0,0 +1,191 @@
+      expect(badge).toHaveClass('bg-error');
+    });
+
+    it('applies exams variant classes', () => {
+      render(<Badge variant="exams" action="exam1">Exam 1</Badge>);
+      const badge = screen.getByText('Exam 1');
+      expect(badge).toHaveClass('bg-exame-1');
+      expect(badge).toHaveClass('text-[#145B8F]');
+    });
+
+    it('applies resultStatus variant classes', () => {
+      render(<Badge variant="resultStatus" action="positive">Positive</Badge>);
```

ESLint

[error] Replace `<Badge·variant="resultStatus"·action="positive">Positive</Badge>` with `⏎········<Badge·variant="resultStatus"·action="positive">⏎··········Positive⏎········</Badge>⏎······`

(prettier/prettier)